beacon-chain/sync/backfill/batch.go

```diff
@@ -101,14 +112,21 @@ func (b batch) ensureParent(expected [32]byte) error {
 	return nil
 }
 
-func (b batch) request() *eth.BeaconBlocksByRangeRequest {
+func (b batch) blockRequest() *eth.BeaconBlocksByRangeRequest {
 	return &eth.BeaconBlocksByRangeRequest{
 		StartSlot: b.begin,
 		Count:     uint64(b.end - b.begin),
 		Step:      1,
 	}
 }
 
+func (b batch) blobRequest() *eth.BlobSidecarsByRangeRequest {
+	return &eth.BlobSidecarsByRangeRequest{
+		StartSlot: b.begin,
+		Count:     uint64(b.end - b.begin),
```

Comment:
Is `b.end > b.begin` requirement verified somewhere?

Reply:
This should be implicitly done in the `before` method of the batcher